normandy/bundler/models.py

```diff
@@ -14,3 +16,14 @@ def ids(self):
     def hash(self):
         recipe_hashes = '_'.join([recipe.content_hash for recipe in self.recipes])
         return hashlib.sha1(recipe_hashes.encode('utf-8')).hexdigest()
+
+    def __iter__(self):
+        for recipe in self.recipes:
+            yield recipe
```

Comment:
I think this would be better as `yield from self.recipes`. (yay Python 3!) Or maybe `return iter(self.recipes)`.